### PR TITLE
Update for PSR-2 standard

### PR DESCRIPTION
**Pull request by svpernova09**
Few of the recent PRs coming into Homestead are mix and matching styles, I ran [PHP-CS-Fixer](https://github.com/FriendsOfPHP/PHP-CS-Fixer) and this is the resulting changes.

### CODE REVIEW

File: homestead [hunk 1]
```diff
@@ -21,18 +21,18 @@ function homestead_path()
...
-$app->add(new Laravel\Homestead\MakeCommand);
-$app->add(new Laravel\Homestead\ProvisionCommand);
-$app->add(new Laravel\Homestead\ResumeCommand);
-$app->add(new Laravel\Homestead\RunCommand);
-$app->add(new Laravel\Homestead\UpCommand);
-$app->add(new Laravel\Homestead\UpdateCommand);
-$app->add(new Laravel\Homestead\SshCommand);
-$app->add(new Laravel\Homestead\StatusCommand);
-$app->add(new Laravel\Homestead\SuspendCommand);
+$app->add(new Laravel\Homestead\DestroyCommand());
```
Comment (GrahamCampbell) [inline homestead]: These changes are NOT PSR-2.

File: src/DestroyCommand.php [hunk 2]
```diff
@@ -21,9 +21,8 @@ protected function configure()
     /**
      * Execute the command.
      *
-     * @param  \Symfony\Component\Console\Input\InputInterface  $input
-     * @param  \Symfony\Component\Console\Output\OutputInterface  $output
-     * @return void
+     * @param \Symfony\Component\Console\Input\InputInterface   $input
```
Comment (GrahamCampbell) [inline src/DestroyCommand.php]: These changes are NOT PSR-2.

File: src/DestroyCommand.php [hunk 3]
```diff
@@ -21,9 +21,8 @@ protected function configure()
     /**
      * Execute the command.
      *
-     * @param  \Symfony\Component\Console\Input\InputInterface  $input
-     * @param  \Symfony\Component\Console\Output\OutputInterface  $output
-     * @return void
```
Comment (GrahamCampbell) [inline src/DestroyCommand.php]: You MUST keep all the return void phpdoc.

**Comment by GrahamCampbell** [comment]
@svpernova09 We actually have our own ruleset for php-cs-fixer. Check out our framework repo.

File: src/DestroyCommand.php (hunk 3 above)
Comment (svpernova09) [inline src/DestroyCommand.php]: @GrahamCampbell I used .php_cs from laravel/framework and it's still removed all the ``` @return void``` lines. Ideas?

Comment (GrahamCampbell) [inline src/DestroyCommand.php]: You probably ran it with the symfony level by mistake.

Comment (svpernova09) [inline src/DestroyCommand.php]: Not sure, I copied ```.php_cs``` from framework to my homestead folder and
ran ```php-cs-fixer fix src```

On Sun, Jul 5, 2015 at 2:19 PM, Graham Campbell <notifications@github.com>
wrote:

> In src/DestroyCommand.php
> <https://github.com/laravel/homestead/pull/226#discussion_r33897973>:
>
> > @@ -21,9 +21,8 @@ protected function configure()
> >      /**
> >       * Execute the command.
> >       *
> > -     * @param  \Symfony\Component\Console\Input\InputInterface  $input
> > -     * @param  \Symfony\Component\Console\Output\OutputInterface  $output
> > -     * @return void
>
> You probably ran it with the symfony level by mistake.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/laravel/homestead/pull/226/files#r33897973>.
>



-- 
- Joe Ferguson
MemphisTechnology.org
MemphisPHP.org
HACKMemphis.com

Comment (GrahamCampbell) [inline src/DestroyCommand.php]: Passing any parameters to php-cs-fixer makes it avoid using the config file. Don't pass `src`.

Comment (svpernova09) [inline src/DestroyCommand.php]: Running it from package root yields no changes, which seems inconsistent
due to the MakeCommand.php having namespace on a different line than other
commands, but it must be something in the .php_cs allowing it so I digress.

On Sun, Jul 5, 2015 at 2:22 PM, Graham Campbell <notifications@github.com>
wrote:

> In src/DestroyCommand.php
> <https://github.com/laravel/homestead/pull/226#discussion_r33898003>:
>
> > @@ -21,9 +21,8 @@ protected function configure()
> >      /**
> >       * Execute the command.
> >       *
> > -     * @param  \Symfony\Component\Console\Input\InputInterface  $input
> > -     * @param  \Symfony\Component\Console\Output\OutputInterface  $output
> > -     * @return void
>
> Passing any parameters to php-cs-fixer makes it avoid using the config
> file. Don't pass src.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/laravel/homestead/pull/226/files#r33898003>.
>



-- 
- Joe Ferguson
MemphisTechnology.org
MemphisPHP.org
HACKMemphis.com

**Pull request closed without merging**